### ISSUE TITLE
KeyError: 'created_at'

### ISSUE THREAD

**Issue by user7590831** [Gertje823/Vinted-Scraper#19]
I'm getting this error when running `python3 scraper.py`

```
Creation of the directory failed or the folder already exists 
Traceback (most recent call last):
  File "/scraper.py", line 596, in <module>
    download_vinted_data(userids, session)
  File "/scraper.py", line 162, in download_vinted_data
    created_at = data['created_at']
KeyError: 'created_at'
```

Could this be due to a API change?

**Comment by TheDevilsMine** [comment]
Same problem here

**Comment by user7590831** [comment]
Looks like they removed it from the payload.
I replaced `created_at = data['created_at']` with `created_at = ""` as a workaround at line 162.

**Comment by TheDevilsMine** [comment]
Same error

**Comment by EmperorMagusCaligula** [comment]
Same issue! I modified line 162 as explained and it works again!

**Comment by TheDevilsMine** [comment]
I got this error now:
Traceback (most recent call last):
  File "<file folder>", line 595, in <module>
    session = vinted_session()
  File "<File folder>", line 61, in vinted_session
    csrfToken = req.text.split('<meta name="csrf-token" content="')[1].split('"')[0]
IndexError: list index out of range

Anyone any idea?

**Comment by TheDevilsMine** [comment]
It works, but not in PyCharm

**Comment by Gertje823** [comment]
As @user7590831 mentioned, it looks like they removed the field from the api response. I updated the script so it will now check if there is a `created_at` value and else ignore it. I also encounterd some Cloudflare errors. It seems they also changed something. So I switched from [cfscrape](https://github.com/Anorov/cloudflare-scrape) to [cloudscraper](https://github.com/VeNoMouS/cloudscraper) I recommend running `pip3 install -r requirements.txt` again, before running the updated script.
